pages/docs/reference/using-gradle.md

```diff
@@ -583,7 +583,7 @@ It is also possible to configure all Kotlin compilation tasks in the project:
 <div class="sample" markdown="1" mode="groovy" theme="idea" data-lang="groovy">
 
 ```groovy
-tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile::class.java).all {
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile.class).all {
```

Comment:
FYI: you can even remove the .class
However the one currently listed in the docs definitely doesn't work in Android Studio.